pages/_language/events/_event/index.vue

```diff
@@ -139,6 +148,19 @@
                     }
                   }
                 }
+                ... on ColofonBlockRecord {
+                  _modelApiKey
+                  id
+                  title
+                  titleColor
+                  showWaveMarker
+                  body
+                  logos {
+                    id
+                    url(imgixParams: {auto: compress, h: "100"})
```

Comment:
**Suggestion**: I'd rather have the full URL here, and add transformations in the template, so data and presentation stay separated.